lib/PuppeteerSharp.Tests/TestConstants.cs

```diff
@@ -17,7 +17,13 @@ public static class TestConstants
 
         public static readonly DeviceDescriptor IPhone = DeviceDescriptors.Get(DeviceDescriptorName.IPhone6);
         public static readonly DeviceDescriptor IPhone6Landscape = DeviceDescriptors.Get(DeviceDescriptorName.IPhone6Landscape);
-        
+
+        public static readonly string NestedFramesDumpResult = @"http://127.0.0.1:<PORT>/frames/nested-frames.html
```

Comment:
maybe create a class `GoldenUtils` that will contain all of the golden files?

Reply:
I'm not so sure yet. That golden guy is used to compare files. They used that tool to compare this dump and I think it makes no sense for them to do that.